3rdparty/haskell/BUILD.data-default

```diff
@@ -8,23 +8,19 @@ load("@ai_formation_hazel//:hazel.bzl", "hazel_library")
 haskell_library(
   name = "data-default",
   visibility = ["//visibility:public"],
-  generator_name = "data-default",
```

Comment:
This was just leftover junk that I forgot to remove when I initially added this hack.